timetemp.py

```diff
@@ -27,7 +27,8 @@
 
 # Fetch weather data from Yahoo!, parse resulting XML
 dom = parseString(urllib.urlopen(
-        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())
+#        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())
+        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())        
```

Comment:
My bad.  I have updated the files with the correct code changes in my branch and re-issued the pull requests.  They should update the code to use the YQL format for the Yahoo weather API.